# ====== DO NOT MODIFY! AUTOGENERATED FROM AN ACTION DEFINITION ======
# Define the result
geometry_msgs/PoseStamped base_poses
arm_navigation_msgs/MotionPlanRequest arm_poses
string planner_service_name
string state
